certify-core/src/main/java/io/mosip/certify/core/dto/ProofKeyCodeExchange.java

```diff
@@ -0,0 +1,46 @@
+package io.mosip.certify.core.dto;
```

Comment:
Is this class required? as the PKCE check is done by the OAuth server and not in the certify-service

Reply:
removed this class